index.html

```diff
@@ -4,15 +4,71 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>AJAX Project</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css" integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
   <link rel="icon" href="images/favicon.ico">
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-  <header>
+  <header class="header">
+    <div class="header-container">
+      <button class="favorites-button">Favorites</button>
+      <button class="new-catch-button">New Catch</button>
+      <button class="library-button">Library</button>
+    </div>
   </header>
   <main>
+    <div data-view="carousel" class="carousel hidden">
+      <div class="left-button">
+        <i class="fa-solid fa-angle-left fa-10x"></i>
+      </div>
+      <div class="favorite-pokemon">
+        <div class="pokemon-image">
+          <img class="favorite-image" src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/shiny/25.png" alt="pikachu">
+        </div>
+        <div class="pokemon-info">
+          <h1 class="main-text carousel-pokemon">Pikachu</h1>
+          <h1 class="secondary-text carousel-name">Bolt</h1>
+          <h1 class="main-text">Encounters</h1>
+          <h1 class="secondary-text carousel-encounters">5890</h1>
```

Comment:
Semantically, there should only be one `<h1>` on a page. If you need something similarly-styled, use custom CSS to get it there.